contentcuration/contentcuration/settings.py

```diff
@@ -40,7 +40,7 @@
 WEBPACK_LOADER = {
     'DEFAULT': {
         # trailing empty string to include trailing /
-        'BUNDLE_DIR_NAME': os.path.join('js', 'bundles', ''),
+        'BUNDLE_DIR_NAME': os.path.join('studio', ''),
```

Comment:
Bundles in production env weren't found - their URLs pointed to "/js/bundles" but I suppose that after this https://github.com/learningequality/studio/pull/1487/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR22 change it should be "/studio".

Everything worked well in dev env as everything seems to be served from "/dist".